build_teamcity_windows.ps1

```diff
@@ -100,6 +100,10 @@ If ( $LASTEXITCODE -ne 0 ) {
 _scope_closed "setup"
 
 release create $TAG $TAG $tree_variants
+if($LASTEXITCODE) {
```

Comment:
Is there a reason for this being structured this was as opposed to the if($LASTEXITCODE -ne 0) format used to test other calls in the script?

Reply:
Thank you for the clarification and willingness to update your patch.
Re-reviewed and approved.